src/bodywork/workflow_execution.py

```diff
@@ -400,3 +399,17 @@ def _remove_readonly(func: Any, path: Any, exc_info: Any) -> None:
         func(path)
     else:
         raise Exception
+
+
+def _ping_usage_stats_server() -> None:
+    """Pings the usage stats server."""
+    try:
+        session = requests.Session()
+        session.mount(
+            USAGE_STATS_SERVER_URL, requests.adapters.HTTPAdapter(max_retries=0)
+        )
+        response = session.get(USAGE_STATS_SERVER_URL)
+        if not response.ok:
+            _log.warning("Unable to contact usage stats server")
```

Comment:
Should we not keep schtum here?

Reply:
I wasn't sure about this either...could go log.info or just remove it. What do you think?

Reply:
I reckon we should use `log.info` for these.

I was about to say, "Log nothing!" But this is probably a bit dishonest and makes it hard to debug, as well.

Reply:
I agree, log.info sounds like a good compromise. Also don't want to set anyone's logging alarms off with a warning for this.